### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by Horttcore** [wp-cli/profile-command#135]
Can't install via terminal

Same problem as the doctor command.
Could this be a local problem?

```wp package install wp-cli/profile-command
Installing package wp-cli/profile-command (dev-master)
Updating /Users/ralfhortt/.wp-cli/packages/composer.json to require the package...
Using Composer to install the package...
---
Loading composer repositories with package information
Updating dependencies
Resolving dependencies through SAT
Dependency resolution completed in 0.345 seconds
Your requirements could not be resolved to an installable set of packages.
Problem 1
    - The requested package runcommand/profile could not be found in any version, there may be a typo in the package name.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see https://getcomposer.org/doc/04-schema.md#minimum-stability for more details.

Read https://getcomposer.org/doc/articles/troubleshooting.md for further common problems.
---
Error: Package installation failed (Composer return code 2). Reverted composer.json```

**Comment by Horttcore** [comment]
There was a dead entry in my wp cli composer json file, that blocked installation.

**Comment by lvl99** [comment]
I had a similar issue. I did this:
```bash
  $ wp cli update
```

Then...
```bash
  $ wp package install git@github.com:wp-cli/profile-command.git
```

But when doing `wp help` the `profile` command wouldn't show up.

I navigated to `~/.wp-cli/packages` to look at the composer.json which had the `require` and `repositories` entries for `profile-command` in it, but I had to run a `composer update` in that folder to get the package to download and install and then I could do all my `wp profile ...` commands.
